cli/tests/services.bats

```diff
@@ -63,42 +69,171 @@ teardown() {
 @test "process-compose can run generated config file" {
   export FLOX_FEATURES_SERVICES=true
   "$FLOX_BIN" init
-  manifest_file="${TESTS_DIR}/services/touch_file/manifest.toml"
-  run "$FLOX_BIN" edit -f "$manifest_file"
+  run "$FLOX_BIN" edit -f "${TESTS_DIR}/services/touch_file.toml"
   assert_success
-  run bash "${TESTS_DIR}/services/touch_file/check_service_ran.sh"
+  run "$FLOX_BIN" activate -- bash <(cat <<'EOF'
+    source "${TESTS_DIR}/services/start_and_cleanup.sh"
+    echo "looking for file"
+    [ -e hello.txt ]
+    echo "found it"
+EOF
+)
   assert_success
 }
 
-@test "'flox activate' with feature flag does not start services" {
-  export FLOX_FEATURES_SERVICES=true
+@test "'flox activate -s' error without feature flag" {
+  export FLOX_FEATURES_SERVICES=false
   "$FLOX_BIN" init
-  manifest_file="${TESTS_DIR}/services/touch_file/manifest.toml"
+  manifest_file="${TESTS_DIR}/services/touch_file.toml"
   run "$FLOX_BIN" edit -f "$manifest_file"
   assert_success
-  "$FLOX_BIN" activate -- true
-  run [ -e hello.txt ]
+  unset output
+  run "$FLOX_BIN" activate -s
   assert_failure
+  assert_output --partial "Services are not enabled in this environment"
 }
 
-@test "'flox activate -s' starts services" {
+# ---------------------------------------------------------------------------- #
+
+# bats test_tags=services:stop
+@test "stop: can't be used without feature flag" {
+  run "$FLOX_BIN" services stop
+  assert_failure
+  assert_output "❌ ERROR: services are not enabled"
+}
+
+# bats test_tags=services:stop
+@test "stop: errors if a service doesn't exist" {
   export FLOX_FEATURES_SERVICES=true
-  "$FLOX_BIN" init
-  manifest_file="${TESTS_DIR}/services/touch_file/manifest.toml"
-  run "$FLOX_BIN" edit -f "$manifest_file"
+  setup_sleeping_services
+
+  run "$FLOX_BIN" activate -- bash <(cat <<'EOF'
+    source "${TESTS_DIR}/services/start_and_cleanup.sh"
+    "$FLOX_BIN" services stop invalid
```

Comment:
nonblocking: if we aren't going to require `stop` to run inside an activation, the tests would be less clunky if we don't do the whole heredoc shenanigans

Reply:
I agree, but I'd like to delay that cleanup a little bit. I feel like our test suite has changed tactics on every PR for service management and I'd like to reduce the churn there until we settle on exactly how we want to test some of this.